packages/apps/contracts/PreimageApp.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity 0.5.8;
+pragma experimental "ABIEncoderV2";
+
+import "@counterfactual/contracts/contracts/interfaces/CounterfactualApp.sol";
+
+
+contract PreimageApp {
```

Comment:
`PreImageApp` (capitalize the letter "I")

Reply:
I don't agree - the spaced out version should be "preimage app" not "pre image app"